tests/integration/API/PostWork.test.ts

```diff
@@ -0,0 +1,85 @@
+/* tslint:disable:no-relative-imports */
+import { pickBy } from 'ramda'
+import { describe } from 'riteway'
+
+import { ABraveAndStartlingTruth } from '../../helpers/Claims'
+import { delay, runtimeId, setUpServerAndDb } from '../../helpers/utils'
+import { getWork, postWork } from '../../helpers/works'
+
+const PREFIX = `test-functional-node-poet-${runtimeId()}`
+const NODE_PORT = '28081'
+const getWorkFromNode = getWork(NODE_PORT)
+const postWorkToNode = postWork(NODE_PORT)
+
+const getClaimWithoutTimestamp = (claim: any) => pickBy((v: any, k: string) => k !== 'timestamp', claim)
+
+describe('POST /works', async assert => {
+  // Setup Mongodb and the app server
+  const { db, server } = await setUpServerAndDb({ PREFIX, NODE_PORT })
+
+  {
+    const response = await postWorkToNode(ABraveAndStartlingTruth)
+    const body = await response.text()
+
+    assert({
+      given: 'POST /works with a signed verifiable claim',
+      should: 'return 202',
+      actual: response.status,
+      expected: 202,
+    })
+
+    assert({
+      given: 'POST /works with a signed verifiable claim',
+      should: 'return an empty body',
+      actual: body,
+      expected: '',
+    })
+
+    await delay(5000)
+
+    const retrievedResponse = await getWorkFromNode(ABraveAndStartlingTruth.id)
+    const retrievedClaim = getClaimWithoutTimestamp(await retrievedResponse.json())
+
+    assert({
+      given: 'POST /works with a signed verifiable claim',
```

Comment:
How about this wording?
given: 'An existing work with a signed verifiable claim'

Reply:
It's not exactly that. For example, is it an existing claim if it hasn't be sent to node yet? I should remove the signed verifiable claim portion, as this iteration is not following the standard yet.

Reply:
oops nm. lol

Reply:
How abut `given: a work posted to node`?